api/Onboard.go

```diff
@@ -0,0 +1,50 @@
+package api
+
+import (
+	"fmt"
+	// "path/filepath"
+	// "github.com/operate-first/opfcli/constants"
+	// "github.com/operate-first/opfcli/utils"
+	"github.com/operate-first/opfcli/models"
+	"strings"
+)
+
+func (api *API) Onboard(path string, projectDisplayName string, disableLimitrange bool) error {
+	onboardRequest, err := models.OnboardRequestFromYAMLPath(path)
+
+	fmt.Printf("onboardRequest, %v\n", onboardRequest)
```

Comment:
This looks like a debugging statement. Should it be removed or replaced with `log.Debugf`?